application/backend/src/main/resources/apps/typerefinery/components/content/barchart/barchart.html

```diff
@@ -0,0 +1,4 @@
+<sly data-sly-use.model="io.typerefinery.websight.models.components.BarChart"
+     data-sly-use.template="./template.html"
+     data-sly-call="${template.barchart @ model=model}">
```

Comment:
Please follow the camel case naming convention.

barchart to barChart

Reply:
change completed.